lib/UtapiClient.js

```diff
@@ -898,6 +898,28 @@ class UtapiClient {
             return undefined;
         });
     }
+
+    /**
+     * Get storage used by bucket/account/user/service
+     * @param {object} params - params for the metrics
+     * @param {string} [params.bucket] - (optional) bucket name
+     * @param {string} [params.accountId] - (optional) account canonical ID
+     * @param {string} [params.userId] - (optional) account ID
+     * @param {string} [params.level] - (required) level of granularity
+     * @param {string} [params.service] - (required) service name (s3 for ex.)
+     * @param {object} log - Werelogs request logger
+     * @param {callback} callback - callback to call
+     * @return {undefined}
+     */
+    getStorageUtilized(params, log, callback) {
+        const key = generateCounter(params, 'storageUtilizedCounter');
```

Comment:
Is there no need for a "time window" for this request ? (I thought that everything was driven through time windows in utapi ?).
Otherwise code & doc look okay

Reply:
Since quotas need current usage, this responds with the global counter which represents the storage used at the moment.